tools/release/main.go

```diff
@@ -116,8 +116,8 @@ func generateFile(name, version string) error {
 		TutorialLink:         tutorialLink,
 		Platform: []Platform{
 			*newPlatform(packageName, version, "x86_64", "linux", "Linux (x86_64)", []string{"tar.gz"}),
-			*newPlatform(packageName, version, "x86_64", "linux", "Debian 9, 10 / Ubuntu 18.04, 20.04", []string{"deb"}),
```

Comment:
[q] are we also removing Debian 9 support? if so, do we need any extra actions taken for this?

Reply:
yes, and extra actions done, the docker files were renamed and updated to 11